src/rss-diff.js

```diff
@@ -1,5 +1,25 @@
-function rssNDiff(itemBefore, itemAfter, field) {
-    let delta
+function rssDiff(itemBefore, itemAfter, field) {
+    let beforeMap = {}
+    let out = []
+    let indicies = []
+
+    // Create a map of field to index for first array
+    for (let i = 0; i < itemBefore.length; i++) {
+        beforeMap[itemBefore[i][field]] = i
+    }
+
+    // If element in second array does not exist in previous map, store its index
+    for (let i = 0; i < itemAfter.length; i++) {
+        if (!(itemAfter[i][field] in beforeMap)) {
```

Comment:
As far as I know Pypi explicitly *does not* allow packages to be removed (to avoid the whole npm fiasco), I'll have to look into how maven will handle it though.